scripts/bump-version.sh

```diff
@@ -21,10 +21,17 @@ cargo update -p sentry-cli
 
 # Do not tag and commit changes made by "npm version"
 export npm_config_git_tag_version=false
+
+# Bump main sentry cli npm package
 npm version "${TARGET}"
 
+# Bump the binary npm distributions
 for dir in $SCRIPT_DIR/../npm-binary-distributions/*; do
     cd $dir
     npm version "${TARGET}"
     cd -
 done
+
+# Update the optional deps in the main cli npm package
+# Requires jq to be installed - should be installed ootb on github runners
+jq '.optionalDependencies |= map_values("'"${TARGET}"'")' $SCRIPT_DIR/../package.json > package.json.tmp && mv package.json.tmp $SCRIPT_DIR/../package.json
```

Comment:
tbh I'd rather we convert this to a node script but 🤷

Reply:
Yeah I also feel conflicted about this, at the same time it felt weird to either call node from this script, or alternatively it felt also weird converting this script into node bc we run cargo & perl 💀